project/Project.scala

```diff
@@ -547,9 +549,57 @@ object Zipkin extends Build {
     receiverScribe, zookeeper
   )
 
-  lazy val prod = Project(
-    id = "zipkin-prod",
-    base = file("zipkin-prod"),
+  lazy val prodWeb = Project(
+    id = "zipkin-prod-web",
+    base = file("zipkin-prod-web"),
+    settings = defaultSettings ++ assemblySettings
+  ).settings(
+      mergeStrategy in assembly <<= (mergeStrategy in assembly) { (old) =>
+      {
+        case PathList("org", xs @ _*) => MergeStrategy.first
+        case PathList("com", xs @ _*) => MergeStrategy.first
+        case "BUILD" => MergeStrategy.first
+        case PathList(ps @_*) if ps.last == "package-info.class" => MergeStrategy.discard
+        case x => old(x)
+      }
+      },
+      libraryDependencies ++= Seq(
+        finagle("zipkin"),
+        finagle("stats"),
+        twitterServer
+      )
+    ).dependsOn(
+      web, query, cassandra, zookeeper
+    )
+
+  lazy val prodReceiver = Project(
```

Comment:
`zipkin-collector-service` is very similar to this, although the main file needs modification to use kafka instead (https://github.com/twitter/zipkin/blob/29e0f90a8d19a6c80eaf0e7504f72058cd05b7a6/doc/kafka-receiver.md), so I'm ok with keeping this.